src/imu/sensors/imu_sensors/general.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef SRC_IMU_MEASUREMENT_SENSOR_HPP_
```

Comment:
general.hpp is a dangerous file name and could conflict with another file

Reply:
Renamed to imu_general